doc/submit-a-report.rst

```diff
@@ -0,0 +1,100 @@
+
+#. **If Spyder presents an error dialog** you can submit an issue directly from it.
+   You will have to fill out a title for your issue, specify the steps that lead to this problem and click :guilabel:`submit to Github`.
+   This will prefill an error report with your environment details, key versions and dependencies and automatically insert the error/traceback for you.
+
+   .. image:: images/submit-a-report/submit-a-report-error-dialog.png
+      :alt: Github page on Spyder's issue tracker
+      :width: 500px
+
+#. **If Spyder opens and your issue does not involve an error dialog**, the best way to do so is to simply select :guilabel:`Report issue` from the :guilabel:`Help` menu, which manually brings up the issue report form and fills in the key information about your Spyder installation.
+   Describe the issue you're experiencing (including any error/traceback information) along with a descriptive title, and click :guilabel:`Submit to Github`.
+
+   .. image:: images/submit-a-report/submit-a-report-issue.png
+      :alt: Github page on Spyder's issue tracker
+
+#. **If Spyder won't launch**, you can submit a report manually at our issues page on `Github`_.
+   Unlike the above, you'll need to manually provide the versions of everything (Spyder, Python, OS, Qt/PyQt, Anaconda, and Spyder's dependencies) as listed in the error report template; see below for more on that.
+
+   .. image:: images/submit-a-report/submit-a-report-issue-template.png
+      :alt: Github page on Spyder's issue template
+
+.. _GitHub: https://github.com/spyder-ide/spyder/issues
+
+Once you submit your report, our team will try to get back to you as soon as possible, often within 24 hours or less, to try to help you fix it.
+
+
+What to include in your report
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Please include as much as possible of the following in your report** to maximize your chances of getting relevant help and our ability to diagnose, reproduce and solve your issue.
+
+The key items, in order of priority:
+
+* The full, **complete error message or traceback** copy/pasted or automatically entered exactly as displayed by Spyder:
+
+    - Auto-generated reports directly from the error dialog should include this automatically, but double check to make sure.
+    - You can copy and paste this from the the :guilabel:`Show Details` section of the error dialog.
+    - If not present, or a dialog is not displayed, you can also find it printed to Spyder's :guilabel:`Internal Console`, located under the :guilabel:`View` menu at :menuselection:`Panes --> Internal Console`.
+    - If you prefer, or if Spyder won't start, you can start Spyder from your command line (or Anaconda prompt on windows) with ``spyder`` and copy the output printed there.
+
+   .. note::
+
+      *If you are reporting a specific behavior* rather than an error, or the message does not fully explain what occurs, please *describe in detail what actually happened, and what you expected Spyder to do*.
```

Comment:
```suggestion
      If you are reporting a specific behavior rather than an error, or the message does not fully explain what occurs, please describe in detail what actually happened, and what you expected Spyder to do.
```

I think the long sentences in italics make this text harder to read.

Reply:
Sure, I guess so